### PR TITLE
Create architecture dissertation work type

### PR DESCRIPTION
**Pull request by laritakr**
Create new work type, based on ETD, with addition of a dissertation type. 

Type is metadata content only and does not behave as type in document... i.e. it does not change the metadata fields for the work subtype. It does get used as work type for displays and faceting. 

https://jira.library.nd.edu/browse/DLTP-1748

### CODE REVIEW

File: config/work_type_policy_rules.yml [hunk 1]
```diff
@@ -8,10 +8,12 @@ CatholicDocument:
   open: all
 Dataset:
   open: all
+Dissertation:
+  open: all
 Document:
   open: all
 Etd:
-  open: nobody
+  open: all
```
Comment (jeremyf) [inline config/work_type_policy_rules.yml]: Can you tell me more about this change?

Comment (laritakr) [reply config/work_type_policy_rules.yml]: This only affects running it locally, and I changed it so I could compare ETD to my new dissertation type. In production, it is controlled by a group id. I didn't change anything in any of the staging or production environments.

Comment (jeremyf) [reply config/work_type_policy_rules.yml]: Cool. That sounds like a great change.

File: app/assets/stylesheets/curate.scss [hunk 2]
```diff
@@ -28,6 +28,6 @@
 @import 'style/theme';
 @import 'style/typography';
 
-.negetive-left-margin {
+.negative-left-margin {
```
Comment (jeremyf) [inline app/assets/stylesheets/curate.scss]: Good catch on spelling!

**Pull request merged**